### ISSUE TITLE
Unable to connect to mysql from host/docker

### ISSUE THREAD

**Issue by phommata** [savyasachi9/sarathy#4]
Docker version 20.10.8, build 3967b7d
Mac OS Big Sure 11.6.5 (20G527)

```
docker run -it --rm --privileged \
    -p 9090-9099:9090-9099 \
    -h sarathy --name sarathy \
    -v ${PWD}:/src/user \
    savyasachi9/sarathy:minikube-amd64

```

I have tinkered with forwarding mysql port to an available host port and I am still unable to access mysql database from host.
```
docker exec -it sarathy mysql -h sarathy -u root -proot
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 2003 (HY000): Can't connect to MySQL server on 'sarathy:3306' (111)

```

**Comment by phommata** [comment]
Similar issue on docker. I am unable to connect to local MySQL server.
```
docker@sarathy:~$ mysql -u root -proot
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)
```

**Comment by savyasachi9** [comment]
> Docker version 20.10.8, build 3967b7d Mac OS Big Sure 11.6.5 (20G527)
> 
> ```
> docker run -it --rm --privileged \
>     -p 9090-9099:9090-9099 \
>     -h sarathy --name sarathy \
>     -v ${PWD}:/src/user \
>     savyasachi9/sarathy:minikube-amd64
> ```
> 
> I have tinkered with forwarding mysql port to an available host port and I am still unable to access mysql database from host.
> 
> ```
> docker exec -it sarathy mysql -h sarathy -u root -proot
> mysql: [Warning] Using a password on the command line interface can be insecure.
> ERROR 2003 (HY000): Can't connect to MySQL server on 'sarathy:3306' (111)
> ```

I realized the command published in 'master' branch is not correct, please try this

`docker exec -it --user docker sarathy /bin/bash -c 'mysql -h sarathy -u root -proot'`

also make sure that mysql pod is up & running before trying to use it
```
kubectl get pods
NAME              READY   STATUS    RESTARTS      AGE
default-mysql-0   1/1     Running   1 (56s ago)   13m
```

**Comment by savyasachi9** [comment]
> Similar issue on docker. I am unable to connect to local MySQL server.
> 
> ```
> docker@sarathy:~$ mysql -u root -proot
> mysql: [Warning] Using a password on the command line interface can be insecure.
> ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)
> ```

for this from within sarathy container you have to use hostname 'sarathy'

`docker@sarathy:~$ mysql -h sarathy -u root -proot`

**Comment by phommata** [comment]
I am still having the same issues for both host and docker after pulling the latest master.

**Comment by phommata** [comment]
I was able to find the root cause and troubleshoot with @savyasachi9's help! Thank you! 

`minikube ip` revealed that Minikube was not running.

`minikube start` returned errors that 6GB memory failed to allocate. (base image is possibly exported with 6GB requirement)

`htop` on the container showed that there was less than 6GB available.

I increased the Docker resources to 6.25GB and was able to resolve the issue. I reran: 
`docker run -it --rm --privileged \
    -p 9090-9099:9090-9099 \
    -h sarathy --name sarathy \
    -v ${PWD}:/src/user \
    savyasachi9/sarathy:minikube-amd64`

`k9s` showed that the `mysql` pod was running after waiting a bit for it to start.

I was able to successfully access `mysql` from the container and host!
